docker/Dockerfile.dev

```diff
@@ -0,0 +1,91 @@
+FROM python:3.8-slim as base
+
+# When building locally, these should be set to your UID/GID.  That way, any
+# files written to the $PWD mount will be owned by you.  This is not
+# necessary (or wanted) when building in Concourse.
+ARG UID=2000
+ARG GID=2000
+ARG USER=app
+
+# In case the host user's GID is already in the base image.
+RUN grep -q ":$GID:" /etc/group \
+ || groupadd --gid="$GID" "$USER"
+
+RUN useradd \
+      --home-dir="/home/$USER" \
+      --no-log-init \
+      --create-home \
+      --shell=/bin/bash \
+      --gid=$GID \
+      --uid=$UID \
+      --no-user-group \
+      --non-unique \
+      "$USER"
+
+RUN apt update \
+ && apt -y upgrade \
+ && apt install -y --no-install-recommends \
+      build-essential \
+      libpq-dev \
+      curl \
+      gnupg2 \
+      procps \
+      git
+
+# Install Fake Pebble LE server
+COPY --from=letsencrypt/pebble /usr/bin/pebble /usr/bin/pebble
+COPY --from=letsencrypt/pebble /test/ /test/
+COPY --from=letsencrypt/pebble-challtestsrv /usr/bin/pebble-challtestsrv /usr/bin/pebble-challtestsrv
+
+RUN cp /test/certs/pebble.minica.pem /usr/local/share/ca-certificates/pebble.crt \
+ && update-ca-certificates
+
+# Install Postgresql
+ENV PG_MAJOR=11
```

Comment:
Grab a newer PostgreSQL version too at some point?

Reply:
no. we're testing against the version we have deployed